### ISSUE TITLE
I should be able to follow the quick start installation, without any errors or issues

### ISSUE THREAD

**Issue by maleck13** [Kuadrant/multicluster-gateway-controller#480]
**What**

Follow: https://docs.kuadrant.io/getting-started/ 

Ensure it works as expected. Any issues should be reported as issues linked to this one.

Validate that this works for both AWS and GCP

**Comment by jasonmadigan** [comment]
Picking up

**Comment by jasonmadigan** [comment]
```bash
export MGC_BRANCH=release-0.1
curl https://raw.githubusercontent.com/Kuadrant/multicluster-gateway-controller/$MGC_BRANCH/hack/quickstart-setup.sh | bash
```

**Comment by jasonmadigan** [comment]
AWS path seems to work fine.

Seeing an issue when selecting the `gcp` route - seems to ignore the selection, and creates an AWS ManagedZone instead.

**Comment by jasonmadigan** [comment]
after chatting with @R-Lawton, created a follow-up issue in https://github.com/Kuadrant/multicluster-gateway-controller/issues/553. 

Will continue testing gcp quickstart with a workaround, by removing MGC_AWS_ACCESS_KEY_ID env-var.

**Comment by jasonmadigan** [comment]
Confirming the gcp option for quickstart works fine (zone + secret created fine), with the MGC_AWS_ACCESS_KEY_ID env var removed. Will re-test when fix lands in #553.

**Comment by jasonmadigan** [comment]
Re-tested via 553, all looks good.
